### PR TITLE
Update neofs components

### PR DESCRIPTION
**Pull request by fyrchik**
Signed-off-by: Evgenii Stratonikov <evgeniy@nspcc.ru>

### CODE REVIEW

**Comment by fyrchik** [comment]
@alexvanin @KirillovDenis do gateways need any config changes?

**Comment by alexvanin** [comment]
Yep.  s3/docker-compose: `S3_GW_WALLET=/wallet.json` -> `S3_GW_WALLET_PATH=/wallet.json`

File: services/storage/.storage.env [hunk 1]
```diff
@@ -64,36 +64,26 @@ NEOFS_TREE_ENABLED=True
...
-
-## First shard
-### Metabase config
+NEOFS_STORAGE_SHARD_0_WRITECACHE_ENABLED=false
+NEOFS_STORAGE_SHARD_0_WRITECACHE_PATH=/storage/wc0
 NEOFS_STORAGE_SHARD_0_METABASE_PATH=/storage/meta0
-### Blobstor config
-NEOFS_STORAGE_SHARD_0_BLOBSTOR_PATH=/storage/blobstor0
+NEOFS_STORAGE_SHARD_0_BLOBSTOR_0_TYPE=blobovnicza
+NEOFS_STORAGE_SHARD_0_BLOBSTOR_0_PATH=/storage/blobstor0
```
Comment (carpawell) [inline services/storage/.storage.env:71]: how about changing the path to "blobovniczas0" or smth like that? all the storages are the same now so "blobstor0" says nothing

**Comment by KirillovDenis** [comment]
> Yep. s3/docker-compose: `S3_GW_WALLET=/wallet.json` -> `S3_GW_WALLET_PATH=/wallet.json`

and  
`S3_GW_LISTEN_DOMAINS_0` -> `S3_GW_LISTEN_DOMAINS`

**Review by carpawell — DISMISSED** [review]

File: services/storage/.storage.env (hunk 1 above)
Comment (fyrchik) [reply services/storage/.storage.env]: Fixed

**Comment by fyrchik** [comment]
@KirillovDenis I have looked on master and it is still an array there https://github.com/nspcc-dev/neofs-s3-gw/blob/master/config/config.yaml#L35 , is it expected?

**Comment by KirillovDenis** [comment]
> @KirillovDenis I have looked on master and it is still an array there https://github.com/nspcc-dev/neofs-s3-gw/blob/master/config/config.yaml#L35 , is it expected?

yes

**Review by alexvanin — APPROVED** [review]

**Review by realloc — APPROVED** [review]
LGTM, didn't test

**Comment by alexvanin** [comment]
@fyrchik by the way, can we add some useful changes from nightly in the master as well?

1) https://github.com/nspcc-dev/neofs-dev-env/pull/214/commits/24a9c5c3f3311369b36b4c6bddee2ae89d8ced7d force volume remove so it doesn't produce error if volume is already removed

2) https://github.com/nspcc-dev/neofs-dev-env/pull/213/commits/a53823624ade65cc46d19924861e01ee6ee3f39d remove hardcoded image location from neo-go and debian images.

**Review by KirillovDenis — APPROVED** [review]

**Comment by fyrchik** [comment]
@alexvanin I let's do it in another PR

**Pull request merged**